### PR TITLE
🦋 THREESCALE-9896: Account Settings > Billing > 3scale Invoices (Implement EmptyState)

### PR DESCRIPTION
**Pull request by lvillen**
[THREESCALE-9896: Account Settings > Billing > 3scale Invoices (Implement EmptyState)](ssues.redhat.com/browse/THREESCALE-9896)

**Before**
Captura de pantalla 2024-11-27 a las 17 07 08

**After**
Captura de pantalla 2024-11-27 a las 17 06 46

**Verification steps**
*TBD*

### CODE REVIEW

**Review by jlledom — COMMENTED** [review]
It looks good to me 👍

Do you know how to add invoices to that screen? To verify it works fine when the table is shown

**Comment by lvillen** [comment]
> Do you know how to add invoices to that screen? To verify it works fine when the table is shown

You have to do it through the master portal, you'll likely find some problems doing this but you can find answers to that at the FAQ. I'd be also happy to help you this.

File: app/views/provider/admin/account/invoices/index.html.slim [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        th role="columnheader" scope="col" Download
+    tbody role="rowgroup"
+      - @invoices.each do |invoice|
+        = content_tag_for :tr, invoice, role: 'row' do
+          td role="cell" data-label="ID"
+            = link_to invoice.friendly_id, provider_admin_account_invoice_path(invoice), title: "Show #{invoice.friendly_id}"
+          td role="cell" data-label="Month"
+            = invoice.name
+            - if invoice.current?
+              |  (current)
```
Comment (jlledom) [inline app/views/provider/admin/account/invoices/index.html.slim:28]: Why did you add a second whitespace here?

https://github.com/3scale/porta/compare/956eb2d86a779372f32f5b668b5cc72003b865dd..ca151d2022bdde115e0bd142396d1fefe58acf15#diff-c16614f1a9e3b8512dc4a156403e715dbb7e365689eb96bea7e1698f0564de58R28

**Review by jlledom — APPROVED** [review]

**Review by mayorova — APPROVED** [review]
